crates/re_types/definitions/rerun/datatypes/aabb2d.fbs

```diff
@@ -0,0 +1,23 @@
+include "arrow/attributes.fbs";
+include "cpp/attributes.fbs";
+include "fbs/attributes.fbs";
+include "python/attributes.fbs";
+include "rust/attributes.fbs";
+
+include "./vec2d.fbs";
+
+namespace rerun.datatypes;
+
+// ---
```

Comment:
you seem to be as undecided as me on keeping these clement'isms or not 😄 (you removed it in a different file in this PR but add it here)